### ISSUE TITLE
Categories link leads to a 404

### ISSUE THREAD

**Issue by vanrysss** [gopheracademy/gopheracademy-web#138]
The "Categories" link on the left hand side of the blog points to http://blog.gopheracademy.com/post/ which returns 404.

**Comment by chilts** [comment]
Ah, I just found this too. Glad it's already been reported. It's still 404'ing.

**Comment by nathany** [comment]
Yup, still broken.  http://blog.gopheracademy.com click Categories

**Comment by ahsanulhaque** [comment]
I'll have a look at this in a bit.

**Comment by bep** [comment]
It still is 404 ...

**Comment by nathany** [comment]
It doesn't seem like articles are categorized. I don't know what this page would display, so the link could probably be removed.

**Comment by chilts** [comment]
This seems to be no longer valid due to the new design. Can probably be closed.
